Angular/angular.md

```diff
@@ -0,0 +1,45 @@
+- HTML is a Markup language and follows a well-defined layout where we can bind the data using Angular and readily render an interface with minimal effort.
+
+## What problems does it solve?
+One of the greatest advantages with Angular is that, it is Cross-Platform and Modular. Angular can be used for developing Web Application along with Progressive Web Applications, Desktop Applications and Native Mobile Applications. Angular is also modular as the whole code is divided into "modules" which consists of components, pipes and directives coupled together. These modules are reusable and helps implement "Lazy Loading" which means that one or more features of the application can be loaded anytime.
+
+### Why is it recommended?
+Angular is recommended to learn because of the Model View Controller (MVC) Architecture that it follows which allows the developers to do more while written lesser lines of code. Angular is also pre-built in many existing frameworks like Ionic and Wejmo and hence allows to add attractive User-Interfaces. Angular also supports Single Page Applications which are updated dynamically by communicating with the Backend Server. This makes Angular worth learning.
+
+### Examples of Websites using Angular
+- [Google About](https://about.google/)
+- [Github Community Forum](https://github.community/)
+- [Microsoft Office Home](https://www.office.com/apps?auth=2)
+- [Forbes](https://www.forbes.com/sites/karstenstrauss/2019/01/22/the-most-sustainable-companies-in-2019)
+
+## Description
+Angular describes itself as "One framework. Mobile & desktop." It is a single-page Client Side Web Framework that follows the MVC Architecture to render Single Page Applications (SPA). Angular is a titan in the Cross-Platform Market alongside React and Flutter, and utilizes Document Object Model to render Web Interfaces. The major points with Angular is its robustness and Browser Support which makes Angular a focal point of a variety of libraries and modules which includes routing system, terminal tools and a list of utilities which plays a major role in abstracting many of the development work beneath the hood. 
+- Angular utilizes Hypertext Markup Language (HTML), Cascading Style Sheets (CSS) and Typescript. Angular is an ever-growing technology and is increasingly being used by Developers to create Front-End Interface. 
+- Angular versions are backward-compatible which means that the latest version of the Angular will still work with the older versions. With the proposal of Ivy, which is a next-generation pipeline model to render Angular compilation, the runtime and load-time on slower networks will be enhanced.
+- Angular comes built-in with multiple features like a Template Engine, Form Validation, CLI Tool and finally the RXJS which is used to make HTTP Request from an Angular Application. 
```

Comment:
I would condense this down to the first two sentences. The rest sounds like ad copy.

Sorry if the length or level of detail wasn't clear in the template. I've added more notes so it is more clear it should be short.